Intersect (Core)/GameObjects/UserVariableBase.cs

```diff
@@ -24,55 +26,11 @@ public UserVariableBase()
         //See https://www.ascensiongamedev.com/topic/749-event-text-variables/ for usage info.
         public string TextId { get; set; }
 
-        public VariableDataType DataType { get; set; } = VariableDataType.Boolean;
+        // TODO rename this
+        [Column("DataType")]
+        public VariableDataType Type { get; set; } = VariableDataType.Boolean;
```

Comment:
Commenting here - we should have a followup ticket to rename this at the `IVariableBase` level to "DataType", and then do subsequent column renames on the other variable types. I had to name this back to the terrible "Type" because of the new inheritance,